packages/coding-standard/packages/object-calisthenics/src/Rules/NoSetterClassMethodRule.php

```diff
@@ -9,14 +9,14 @@
 use PhpParser\Node\Stmt\ClassMethod;
 use PHPStan\Analyser\Scope;
 use PHPStan\Rules;
-use PHPStan\Rules\Rule;
+use Symplify\CodingStandard\Rules\AbstractSymplifyRule;
 
 /**
  * @see https://github.com/object-calisthenics/phpcs-calisthenics-rules#9-do-not-use-getters-and-setters
  *
  * @see \Symplify\CodingStandard\ObjectCalisthenics\Tests\Rules\NoSetterClassMethodRule\NoSetterClassMethodRuleTest
  */
-final class NoSetterClassMethodRule implements Rule
+final class NoSetterClassMethodRule extends AbstractSymplifyRule
```

Comment:
@samsonasik Hi, I made some changes for default PHPStan rule we inherit from.

I've decoupled `AbstractSymplifyRule` that handles simple repeated code, like get class name.
Now all Symplify PHPStan rules should inherit from it.

Please check it and extend it, if you have some other repeated code that would be nice to handle in parent method.